src/acsets/acsets.py

```diff
@@ -259,7 +259,7 @@ def __init__(
             ob: create_model(
                 ob.name,
                 **{
-                    prop.name: (Union[self.valtype(prop), None], None)
+                    prop.name: (Union[self.valtype(prop), None], Field(title=ob.title, description=ob.description))
```

Comment:
is it the case that all elements will always be lists? If that's the case, we can use this line to introduce default empty lists.